src/express/commands/setup-datadog.js

```diff
@@ -42,27 +41,17 @@ export async function setupDatadog() {
     doc = await loadDevnetConfig('remote')
   }
 
-  if (doc.cloud.toString() === constants.cloud.GCP) {
-    // not tested datadog setup in GCP
-    console.log('📍Datadog setup currently not supported in GCP')
-    return
-  }
-
-  if (doc.devnetBorHosts.length > 0) {
-    console.log('📍Monitoring the nodes', doc.devnetBorHosts[0])
-  } else {
-    console.log(
-      '📍No nodes to monitor since this command is not yet supported on Erigon devnets, please check your configs! Exiting...'
-    )
-    process.exit(1)
+  // Using heimdall hosts here so it handles both bor and erigon
+  if (doc.devnetHeimdallHosts.length > 0) {
```

Comment:
Makes sense. Thinking tho: what happens if we run the devnet with flag `bor.withoutHeimdall`? Will it still have `devnetHeimdallHosts` available? Maybe we need to double check this case

Reply:
Do we have the option to run a devnet without heimdall?

Reply:
yes, in matic-cli we use that option with command `await shadow(options.shadowFork)`
the flag in bor to achieve that is `--bor.withoutheimdall`
What I am not sure about is the existence of `devnetHeimdallHosts` in case we run express-cli on a shadow fork.
Can you test it?
If `devnetHeimdallHosts` is created also in case of a shadow node, then this change is ok.
Thanks

Reply:
So, in express-cli we first create a devnet and then have an option to shadow fork it afaik, in which config file remains the same. Is there a way to start up a shadow from the start in express-cli, if not then we are good.

Reply:
That's ok then, thanks!